README.md

```diff
@@ -31,18 +31,31 @@ These three APIs can be accessed via the waybackpy either by importing it in a s
 
 ### 🏗 Installation
 
-Using [pip](https://en.wikipedia.org/wiki/Pip_(package_manager)), from [PyPI](https://pypi.org/) (recommended):
+**Using [pip](https://en.wikipedia.org/wiki/Pip_(package_manager)), from [PyPI](https://pypi.org/) (recommended)**:
 
 ```bash
 pip install waybackpy
 ```
 
-Install directly from [this git repository](https://github.com/akamhy/waybackpy) (NOT recommended):
+
+**Using [conda](https://en.wikipedia.org/wiki/Conda_(package_manager)), from [conda-forge](https://conda-forge.org/) (recommended)**:
+
+See also [waybackpy feedstock](https://github.com/conda-forge/waybackpy-feedstock), maintainers are [@rafaelrdealmeida](https://github.com/rafaelrdealmeida/),
+ [@labriunesp](https://github.com/labriunesp/)
+ and [@akamhy](https://github.com/akamhy/).
+
+```bash
+conda install waybackpy
```

Comment:
This works, but I believe it would be worth making it explicit to use the conda-forge channel

```bash
conda install -c conda-forge waybackpy

```

for details see:

- https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/channels.html

- https://anaconda.org/conda-forge/waybackpy